conda-forge.yml

```diff
@@ -6,4 +6,10 @@ bot:
   automerge: true
 build_platform:
   osx_arm64: osx_64
-test: native_and_emulated
\ No newline at end of file
+test: native_and_emulated
+azure:
+  # flag for forcing the building all supported providers
+  force: true
+  # toggle for storing the conda build_artifacts directory (including the
+  # built packages) as an Azure pipeline artifact that can be downloaded
+  store_build_artifacts: true
```

Comment:
You need to be very careful not to leave this in the PR once it gets merged to main.

Reply:
I have removed the azure artifacts. I am now using docker on my computer to build and test the artifacts.